code/species_defs/species_defs.cpp

```diff
@@ -328,6 +328,31 @@ void parse_species_tbl(const char *filename)
 			if (optional_string("$Countermeasure type:"))
 				stuff_string(species->cmeasure_name, F_NAME, NAME_LENGTH);
 
+			if (optional_string("$Custom wingmate formations:")) {
+
+				while (check_for_string("(")) {
+					species->formations.emplace_back();
+					if (stuff_vec3d_list(species->formations.back().data(), MAX_SHIPS_PER_WING - 1) != MAX_SHIPS_PER_WING - 1) {
+						Warning(LOCATION, "A custom formation for species %s did not have 5 positions. Ignoring.", species->species_name);
+						species->formations.pop_back();
+					}
+				}
+			}
+
+			if (optional_string("$Default formation index:")) {
+				if (species->formations.empty()) {
+					Warning(LOCATION, "A default formation index was provided for species %s but no formations are defined.", species->species_name);
+				}
+				else {
+					stuff_int(&species->default_formation);
+
+					if (species->default_formation >= species->formations.size() || species->default_formation <= -2) {
+						Warning(LOCATION, "Invalid default formation index for species %s. Ignoring.", species->species_name);
+						species->default_formation = -1;
+					}
+				}
+			}
+
```

Comment:
There needs to be an `else if (!no_create)` to set `default_formation` to -1.  Alternatively (and this is the better method) set it in the `species_info()` default constructor.